README.md

```diff
@@ -64,11 +47,6 @@ of use:
 
     OPENSHIFT_AEROGEAR_PUSH_IP                         The IP address used to bind JBossAS
     OPENSHIFT_AEROGEAR_PUSH_HTTP_PORT                  The JBossAS listening port
-    OPENSHIFT_AEROGEAR_PUSH_TOKEN_KEY                  The token key for the SimplePush Server
-    OPENSHIFT_AEROGEAR_PUSH_CLUSTER_PORT               
```

Comment:
@kpiwko I *think* they were introduced with the advent of SimplePush.

@fjuma makes sense to keep them or good to remove?